ru.proninyaroslav.libretrack.yaml

```diff
@@ -0,0 +1,99 @@
+app-id: ru.proninyaroslav.libretrack
+runtime: org.gnome.Platform
+runtime-version: '40'
+sdk: org.gnome.Sdk
+command: libretrack
+separate-locales: false
+finish-args:
+  # Required due to being a GUI application
+  - --socket=x11
+  - --socket=fallback-x11
+  # Required to make sure x11 performance is achived on all platforms
+  # At least that's what the legends tell. it might be worth experimenting
+  # with dropping this permission.
+  - --share=ipc
+  # Required for experimental wayland support
+  - --socket=wayland
+  # Required to provide notification functionality
+  - --socket=pulseaudio
+  # As a tracking application, networking is required
+  - --share=network
+  - --device=all
+  # Required for notifications in various desktop environments
+  - --talk-name=org.freedesktop.Notifications
+  - --talk-name=org.kde.StatusNotifierWatcher
+  # Required due to KDE's special way of handling notifications
+  - --own-name=org.kde.StatusNotifierItem-2-1
+  # Required for advanced input methods e.g. writing CJK languages
+  - --talk-name=org.freedesktop.portal.Fcitx
+  # Required for correct work of Flutter packages, which use system DBus
+  # for example https://pub.dev/packages/connectivity_plus
+  - --socket=system-bus
+  # TODO: Temporary fix https://github.com/canonical/dbus.dart/issues/257
+  - --socket=session-bus
```

Comment:
and here too?

Wildcard are allowed.

`--log-session-bus` and `--log-system-bus` are usefull to figure it out.

Reply:
I have given the reason why I can't use `talk-name` at the moment https://github.com/canonical/dbus.dart/issues/257